app/assets/stylesheets/config/_bootstrap_variables.scss

```diff
@@ -6,20 +6,23 @@
 // General style
 $font-family-sans-serif:  $body-font;
 $headings-font-family:    $headers-font;
-$body-bg:                 $light-gray;
+$body-bg:                 white;
 $font-size-base: 1rem;
 
 // Colors
 $body-color: $gray;
-$primary:    $blue;
-$success:    $green;
+$primary:    black;
+$success:    white;
 $info:       $yellow;
 $danger:     $red;
 $warning:    $orange;
 
 // Buttons & inputs' radius
-$border-radius:    2px;
+$border-radius:    5px;
 $border-radius-lg: 2px;
 $border-radius-sm: 2px;
 
 // Override other variables below!
+.btn-success {
+  border: 1px solid black !important;
```

Comment:
Actually dont need this. There's btn-outline-dark in bootstrap, can use that class instead if we need the outline, so add it to the button you need the outline instead of doing this override.

https://getbootstrap.com/docs/4.6/components/buttons/#outline-buttons